docs/components/api.md

```diff
@@ -81,20 +81,104 @@ window.addEventListener('appload', (event) => {
 
 ## Other
 
-- [**Host**](./host-element.md): Host is a functional component that can be used at the root of the render function to set attributes and event listeners to the host element itself.
+The following primitives can be imported from the `@stencil/core` package and used within the lifecycle of a component:
+
+- [**Host**](./host-element.md): `<Host>`, is a functional component that can be used at the root of the render function to set attributes and event listeners to the host element itself. Refer to the [Host Element](./host-element.md) page for usage info.
+
+- **Fragment**: `<Fragment>`, often used via `<>...</>` syntax, lets you group elements without a wrapper node.
+
+  __Type:__ `FunctionalComponent`<br />
+  __Example:__
+  ```ts
+  import { Component, Fragment } from '@stencil/core'
+  @Component({
+    tag: 'cmp-fragment',
+  })
+  export class CmpFragment {
+    render() {
+      return (
+        <>
+          <div>...</div>
+          <div>...</div>
+          <div>...</div>
+        </>
+      );
+    }
+  }
+  ```
 
```

Comment:
It might be worth calling out here that [jsxFragmentFactory](https://www.typescriptlang.org/tsconfig#jsxFragmentFactory) needs to be set to 'Fragment' here for this to compile. Users will need to add the following to their `tsconfig.json` file:
```diff
diff --git a/tsconfig.json b/tsconfig.json
index 09367ce..c0d45e9 100644
--- a/tsconfig.json
+++ b/tsconfig.json
@@ -14,7 +14,8 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "jsx": "react",
-    "jsxFactory": "h"
+    "jsxFactory": "h",
+    "jsxFragmentFactory": "Fragment"
   },
   "include": [
     "src"
```